ci/taos/checker-pr-signed-off-by.sh

```diff
@@ -76,17 +76,17 @@ cibot_report $TOKEN "pending" "TAOS/pr-format-signedoff" "Triggered. The commit
 if [[ $input_result == "success" ]]; then
     # in case of success
     message="Successfully signedoff! This PR includes Signed-off-by: string."
-    cibot_report $TOKEN "success" "TAOS/pr-format-signedoff" "$message" "$REPOSITORY_WEB/pull/$input_pr/commits/$input_commit" "$GITHUB_WEBHOOK_API/statuses/$input_commit"
+    cibot_report $TOKEN "success" "TAOS/pr-format-signedoff" "$message" "${CISERVER}${PRJ_REPO_UPSTREAM}/ci/${dir_commit}/" "$GITHUB_WEBHOOK_API/statuses/$input_commit"
```

Comment:
BTW, It's not a link error.  It is linked to show the **Signed-off-by:....** statement that developer submitted. 

Which better between the existing link and your proposal? :)

Reply:
then, how about this link: https://github.com/nnsuite/TAOS-CI/wiki/TAOS-CI-Plug-in-Modules ?

Reply:
@ohsewon , As we discussed on Nov-01-2018, Please try to submit new module (e.g., pr-format-signed-off.sh) by cloning the existing `pr-forat-nobody.sh`. 
* https://github.com/nnsuite/TAOS-CI/issues/369